### ISSUE TITLE
styles should not be included if browser supports details natively

### ISSUE THREAD

**Issue by ffuubarbuzz** [chemerisuk/better-details-polyfill#2]
Here https://github.com/chemerisuk/better-details-polyfill/blob/master/dist/better-details-polyfill.js#L82 styles should not be included if browser supports details natively

**Comment by chemerisuk** [comment]
@ThiRaBrTNK they should, to normalize the element presentation in all browsers

**Comment by ffuubarbuzz** [comment]
Obviously, it's only your choice, but in my opinion styles normalisation looks like work for normalize.css. better-details-polyfill styles even makes native chrome's details worse, clicking arrow doesn't trigger details switch. Personally, I'd prefer native implementations have their own styles and polyfill styling only polyfill.

**Comment by smohadjer** [comment]
I agree with ThiRaBrTNK. This is a polyfill after all.
